pages/_app.tsx

```diff
@@ -92,52 +65,20 @@ function MyApp({ Component, pageProps }: AppProps) {
             {({ searching }) => {
               return (
                 <AppLayout>
-                  <AppHeader>
-                    <HeaderTitle>
-                      {windowDimensions.width > 550 ? (
-                        <Link href="/" passHref>
-                          👋 Holaplex
-                        </Link>
-                      ) : (
-                        <Link href="/" passHref>
-                          👋
-                        </Link>
-                      )}
-                    </HeaderTitle>
-                    <Space size="large">
-                      <HeaderLinkWrapper active={router.pathname == "/storefronts"}>
-                        <Link href="/storefronts" passHref >
-                          View Stores
-                        </Link>
-                      </HeaderLinkWrapper>
-                      <HeaderLinkWrapper active={false}>
-                        <a
-                          href="https://docs.google.com/document/d/e/2PACX-1vSMDXh4TbZCR70AqA3O3-pRnvWVBSxE5cFM4LcprS_BHa20aRK8xFLDMwv-2YbZydWYQS1DBl2GBFNX/pub"
-                          target="_blank"
-                          rel="noreferrer"
-                        >
-                          FAQ
-                        </a>
-                      </HeaderLinkWrapper>
-                      {windowDimensions.width > 550 && <SocialLinks />}
-                    </Space>
-                  </AppHeader>
+                  {showHeader && <AppHeader />}
```

Comment:
We should always show the header. Please revert this.

Reply:
@kespinola Yes let's touch base with him.  He says he envisions the uploader as a full screen modal in the future that can be kicked off in several places not as its own route.